src/services/refactors/convertParamsToDestructuredObject.ts

```diff
@@ -115,16 +110,21 @@ namespace ts.refactor.convertParamsToDestructuredObject {
             const functionSymbols = map(functionNames, getSymbolTargetAtLocation);
             const classSymbols = map(classNames, getSymbolTargetAtLocation);
             const isConstructor = isConstructorDeclaration(functionDeclaration);
-            const interfaceSymbols = map(functionNames, getSymbolForInterfaceSignature);
+            const contextualSymbols = map(functionNames, (name) => (getSymbolForContextualType(name, checker)));

```

Comment:
*extremely minor, ignore if short on time*
```suggestion
            const contextualSymbols = map(functionNames, name => getSymbolForContextualType(name, checker));
```